test/lsp.py

```diff
@@ -1534,6 +1541,70 @@ def test_custom_includes(self, solc: JsonRpcProcess) -> None:
         self.expect_equal(len(diagnostics), 1, "no diagnostics")
         self.expect_diagnostic(diagnostics[0], code=2018, lineNo=5, startEndColumns=(4, 62))
 
+    def test_remapping_wrong_paramateres(self, solc: JsonRpcProcess) -> None:
```

Comment:
```suggestion
    def test_remapping_wrong_parameters(self, solc: JsonRpcProcess) -> None:
```
Also, something like `test_remapping_wrong_parameter_types` would be more precise.